components/views/save.js

```diff
@@ -1,13 +1,100 @@
 import React from "react";
-import { Text, View } from "react-native";
-import { Standard } from "components/styles/save-style";
+import { Text, View, Image, FlatList, TouchableOpacity } from "react-native";
+import { Chip } from "react-native-elements/dist/buttons/Chip";
+import GradientText from "../colors/gradient-text";
+import Database from "../database";
+import saveStyle from "../styles/save-style";
 
-function SaveScreen() {
-  return (
-    <View style={Standard.container}>
-      <Text>Save!</Text>
+const renderSaveScreen = ({ item }) => (
+  <View style={saveStyle.card}>
+    <Image
+      source={{ uri: item.images[0] }}
+      resizeMode="cover"
+      overflow="hidden"
+      style={saveStyle.image}
+    />
+    <View style={saveStyle.removeBox}>
+      <TouchableOpacity
+        style={saveStyle.remove}
+        onPress={() => remove(item.id)}
+      />
     </View>
-  );
+    <View style={saveStyle.cardContent}>
+      <Text style={saveStyle.name}>{item.name}</Text>
+
+      <View style={saveStyle.chipBox}>
+        <Chip
+          title={
+            <GradientText style={saveStyle.chipText}>
+              {item.breeds.length > 1 ? "Mixed" : item.breeds[0]}
+            </GradientText>
+          }
+          titleStyle={saveStyle.chipText}
+          buttonStyle={saveStyle.chip}
+        />
+        <Chip
+          title={
+            <GradientText style={saveStyle.chipText}>
+              {item.gender}
+            </GradientText>
+          }
+          titleStyle={saveStyle.chipText}
+          buttonStyle={saveStyle.chip}
+        />
+        <Chip
+          title={
+            <GradientText style={saveStyle.chipText}>
+              {new Date().getFullYear() - item.birthDate.year + " y/o"}
+            </GradientText>
+          }
+          titleStyle={saveStyle.chipText}
+          buttonStyle={saveStyle.chip}
+        />
+      </View>
+    </View>
+  </View>
+);
+
+function remove(id) {
+  console.log("Remove " + id);
 }
 
-export default SaveScreen;
+export default class SavedScreen extends React.Component {
```

Comment:
```suggestion
export default function SavedScreen() {
  const [saveData, setSaveData] = useState([]);
  const [fetching, setFetching] = useState(false);

  const getData = () => {
    Database.getItem("data").then((data) => {
      let filtered = data.filter((items) => items.saved);
      setSaveData(filtered);
    });
  }

  const onRefresh = () => {
    setFetching(true);
    getData();
    setFetching(false);
  }

  useEffect(() => {
    getData();
  }, []);

  return (
    <FlatList
      data={saveData}
      extraData={saveData}
      keyExtractor={(item) => item.id.toString()}
      renderItem={renderSaveScreen}
      refreshing={fetching}
      onRefresh={() => onRefresh}
    />
  );
}

```
you can adapt it like this :)